src/Microsoft.CodeAnalysis.FxCopAnalyzers/RulesMissingDocumentation.md

```diff
@@ -2,53 +2,16 @@
-CA1069 | https://docs.microsoft.com/visualstudio/code-quality/ca1069 | Enums values should not be duplicated |
-CA1070 | https://docs.microsoft.com/visualstudio/code-quality/ca1070 | Do not declare event fields as virtual |
-CA1509 | https://docs.microsoft.com/visualstudio/code-quality/ca1509 | Invalid entry in code metrics rule specification file |
 CA1830 | https://docs.microsoft.com/visualstudio/code-quality/ca1830 | Prefer strongly-typed Append and Insert method overloads on StringBuilder. |
 CA1831 | https://docs.microsoft.com/visualstudio/code-quality/ca1831 | Use AsSpan or AsMemory instead of Range-based indexers when appropriate |
 CA1832 | https://docs.microsoft.com/visualstudio/code-quality/ca1832 | Use AsSpan or AsMemory instead of Range-based indexers when appropriate |
 CA1833 | https://docs.microsoft.com/visualstudio/code-quality/ca1833 | Use AsSpan or AsMemory instead of Range-based indexers when appropriate |
 CA1834 | https://docs.microsoft.com/visualstudio/code-quality/ca1834 | Consider using 'StringBuilder.Append(char)' when applicable. |
+CA1835 | https://docs.microsoft.com/visualstudio/code-quality/ca1835 | Prefer the 'Memory'-based overloads for 'ReadAsync' and 'WriteAsync'. |
+CA1836 | https://docs.microsoft.com/visualstudio/code-quality/ca1836 | Prefer IsEmpty over Count |
 CA2008 | https://docs.microsoft.com/visualstudio/code-quality/ca2008 | Do not create tasks without passing a TaskScheduler |
-CA2009 | https://docs.microsoft.com/visualstudio/code-quality/ca2009 | Do not call ToImmutableCollection on an ImmutableCollection value |
-CA2010 | https://docs.microsoft.com/visualstudio/code-quality/ca2010 | Always consume the value returned by methods marked with PreserveSigAttribute |
-CA2011 | https://docs.microsoft.com/visualstudio/code-quality/ca2011 | Avoid infinite recursion |
 CA2012 | https://docs.microsoft.com/visualstudio/code-quality/ca2012 | Use ValueTasks correctly |
-CA2013 | https://docs.microsoft.com/visualstudio/code-quality/ca2013 | Do not use ReferenceEquals with value types |
 CA2014 | https://docs.microsoft.com/visualstudio/code-quality/ca2014 | Do not use stackalloc in loops. |
-CA2015 | https://docs.microsoft.com/visualstudio/code-quality/ca2015 | Do not define finalizers for types derived from MemoryManager<T> |
-CA2244 | https://docs.microsoft.com/visualstudio/code-quality/ca2244 | Do not duplicate indexed element initializations |
-CA2245 | https://docs.microsoft.com/visualstudio/code-quality/ca2245 | Do not assign a property to itself. |
-CA2246 | https://docs.microsoft.com/visualstudio/code-quality/ca2246 | Assigning symbol and its member in the same statement. |
 CA2247 | https://docs.microsoft.com/visualstudio/code-quality/ca2247 | Argument passed to TaskCompletionSource constructor should be TaskCreationOptions enum instead of TaskContinuationOptions enum. |
-CA5359 | https://docs.microsoft.com/visualstudio/code-quality/ca5359 | Do Not Disable Certificate Validation |
```

Comment:
Thanks a lot @LLLXXXCCC for writing documentation for all these security rules!!!